app/jobs/notify_email_job.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+# This job is responsible for sending the notification to the Gov.UK Notify service.
+# It runs as a retryable job to handle the case where the Gov.UK service is temporarily offline.
+class NotifyEmailJob < ApplicationJob
+  queue_as :notifications
+
+  def perform(notification_id:)
+    notification = Notification.emails.kept.includes(:subscription).find(notification_id)
+    return unless notification.subscription.enabled?
+
+    if ENV.fetch('GOVUK_NOTIFY_API_KEY', nil).blank?
+      Rails.logger.error('[NotifyEmailJob] please set the GOVUK_NOTIFY_API_KEY env variable')
```

Comment:
we won't get any notification of this happening - maybe we should be calling Sentry instead?